.storybook/webpack.config.js

```diff
@@ -56,7 +67,7 @@ module.exports = async (webpackConfig) => {
         {
           loader: 'sass-loader',
           options: {
-            additionalData: `@import '${getStylesPath()}';`,
+            additionalData: `${getFontPath()} @import '${getStylesPath()}';`,
```

Comment:
Do we need to do the same thing for `./images` and `$image-path`?

Reply:
Added, because it IS needed 🙂